examples/use-terraformer-instance/README.md

```diff
@@ -0,0 +1,60 @@
+# Using the Terraform instance #
+
+This example code shows how to use the Terraformer code to spin up a
+new instance in the operations subnet of an existing assessment
+environment already created by running `terraform apply` with the
+[cisagov/cool-assessment-terraform](https://github.com/cisagov/cool-assessment-terraform)
+root module.
+
+## Requirements ##
+
+| Name | Version |
+|------|---------|
+| terraform | ~> 0.13.0 |
```

Comment:
To reflect the change in https://github.com/cisagov/cool-assessment-terraform/pull/132/commits/54f4e06f322ebb450a768a9accc5480f2025309f:
```suggestion
| terraform | ~> 0.14.0 |
```